test/util/verifySmartUnpack.ts

```diff
@@ -0,0 +1,49 @@
+import { readArchiveHeaderSync } from '../../src/disk';
+import fs from '../../src/wrapped-fs';
+import * as path from 'path';
+import walk from './walk';
+
+const rootDir = path.resolve(__dirname, '..', '..');
+
+export const verifySmartUnpack = async (asarPath: string) => {
```

Comment:
I don't have a strong opinion, I'm just curious - why `export const` declarations for these functions but `export function` for the ones below?

Reply:
Yeahhh, I'll tidy that up! Had copied some helper code from electron-builder and hadn't modified those. I'll fix it

Reply:
Updated all test functions to be `export const`. I didn't modify src code even though those are `export function`. I personally prefer arrow functions, but am happy to adjust if consistency across src+test is preferred.